src/NServiceBus.NHibernate/TimeoutPersisters/NHibernateTimeoutStorage.cs

```diff
@@ -54,10 +56,18 @@ protected override void Setup(FeatureConfigurationContext context)
                 }
             }
 
+            var timeoutsCleanupExecutionInterval = 
+                context.Settings.GetOrDefault<TimeSpan?>("NHibernate.Timeouts.CleanupExecutionInterval") ?? TimeSpan.FromMinutes(2);
+
+            var timeoutsCleanupQueryPeriod =
+                context.Settings.GetOrDefault<TimeSpan?>("NHibernate.Timeouts.CleanupQueryPeriod") ?? TimeSpan.FromMinutes(5);
```

Comment:
Fair point. I did not want to make query to be 'from the beginning of time until now' but in fact we are doing this type of query on each endpoint start.